packages/i18n/load-i18n/load-i18n.js

```diff
@@ -9,16 +9,16 @@ export default function loadI18n(lang) {
 
   const reactIntlChunkImport = import(/* webpackChunkName: "react-intl-data-[request]" */
   /* webpackMode: "lazy" */
-  `./locale-data/${lang}`);
+  `../locale-data/${lang}`);
```

Comment:
just wanted to change the directory structure so that in both cases, data was one folder down.